.github/workflows/avm.res.app.job.yml

```diff
@@ -0,0 +1,86 @@
+name: "avm.res.app.job"
+
```

Comment:
Blocking comment until MAR file is updated

Reply:
MAR manifest updated; you can resolve this now.